package.json

```diff
@@ -41,6 +41,7 @@
     "metalsmith-serve": "0.0.3",
     "metalsmith-stylus": "1.0.0",
     "ncp": "2.0.0",
+    "node-version-data": "~1.0.0",
```

Comment:
Yes, we are using greenkeeperio here. example: https://github.com/nodejs/new.nodejs.org/pull/279